app/controllers/posts_controller.rb

```diff
@@ -3,10 +3,24 @@ def index
     @posts = Post.all
     @user = User.where(id: params[:user_id]).first
     @comments = Comment.all
+    @like = Like.all
   end
 
   def show
     @post = Post.find(params[:id])
     @comments = Comment.all
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    new_post = Post.new(Title: params[:post][:Title], Text: params[:post][:Text], author: current_user)
+    if new_post.save
+      redirect_to user_posts_path
+    else
+      render :new
+    end
```

Comment:
- [ ] when I tried to create a post using the new page it didn't create any post please make sure to fix that 
<img width="237" alt="Screenshot 2023-02-03 at 15 38 42" src="https://user-images.githubusercontent.com/61790587/216631501-c30e938e-7a48-49ce-a4b6-e93d1f24b739.png">